pom.xml

```diff
@@ -20,6 +20,7 @@
         <!-- Versions -->
         <version.spring.boot>2.4.0</version.spring.boot>
         <version.springdoc.openapi>1.5.1</version.springdoc.openapi>
+        <structured-logging.version>1.5.0-rc2</structured-logging.version>
```

Comment:
Why did you choose this version?

Reply:
Same version that has been set on other repos (BADOs api) and without the need of excluding `logback` to `spring-boot-starter-web` and `structured-logging`